core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -720,10 +733,34 @@ public String interInstanceProtocol() {
     return getString(INTER_INSTANCE_PROTOCOL_CONFIG);
   }
 
+  /**
+   * Gets the inter.instance.listener.name setting.
+   */
+  public String interInstanceListenerName() {
+    return getString(INTER_INSTANCE_LISTENER_NAME_CONFIG);
+  }
+
   public List<String> whitelistHeaders() {
     return getList(INTER_INSTANCE_HEADERS_WHITELIST_CONFIG);
   }
 
+  public Map<String, Object> getOverriddenSslConfigs(NamedURI listener) {
```

Comment:
Can we use `getSslConfig(listener)` implemented in rest-utils for this purpose?
https://github.com/confluentinc/rest-utils/pull/362/files#diff-d8cc169636b9f49499c018cc01dce8e1f4caf3efc3a3c5fbcd74e8fbb3488404R1145